packages/website/src/components/RulesTable/index.tsx

```diff
@@ -191,29 +217,69 @@ export default function RulesTable({
           <RuleFilterCheckBox
             mode={filters.fixable}
             setMode={(newMode): void => changeFilter('fixable', newMode)}
-            label="🔧 fixable"
+            label={`${FIXABLE_EMOJI} fixable`}
           />
           <RuleFilterCheckBox
             mode={filters.suggestions}
             setMode={(newMode): void => changeFilter('suggestions', newMode)}
-            label="💡 has suggestions"
+            label={`${SUGGESTIONS_EMOJI} has suggestions`}
           />
           <RuleFilterCheckBox
             mode={filters.typeInformation}
             setMode={(newMode): void =>
               changeFilter('typeInformation', newMode)
             }
-            label="💭 requires type information"
+            label={`${TYPE_INFORMATION_EMOJI} requires type information`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.extension}
+            setMode={(newMode): void => changeFilter('extension', newMode)}
+            label={`${EXTENSION_RULE_EMOJI} extension rule`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.formatting}
+            setMode={(newMode): void => changeFilter('formatting', newMode)}
+            label={`${FORMATTING_RULE_EMOJI} formatting rule`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.deprecated}
+            setMode={(newMode): void => changeFilter('deprecated', newMode)}
+            label={`${DEPRECATED_RULE_EMOJI} deprecated rule`}
           />
         </ul>
       </div>
       <table className={styles.rulesTable}>
         <thead>
           <tr>
             <th className={styles.ruleCol}>Rule</th>
-            <th className={styles.attrCol}>Config</th>
-            <th className={styles.attrCol}>Fixer</th>
-            <th className={styles.attrCol}>Typed</th>
+            <th className={styles.attrCol}>
+              <button title="The config group that the rule belongs to, if any.">
+                C
+              </button>
```

Comment:
First, I needed to shorten the columns to a single letter to minimize the width of all the columns. (Otherwise, there's very little room for the actual rule name + rule description in the left most column.)

However, once you do that, the columns names are no longer easily identifiable from just their first letter alone, so I needed to add a text-on-hover functionality, and using a button with `title` text seemed to be the easiest way to do that. For example, if I just use normal text with a `title` element, then its non-obvious that it actually has mouseover functionality (since it just looks like normal text).

I'm open to other ideas if you want to workshop this further

Reply:
Makes sense how you got there. The problem with a button is that it's an accessibility issue: buttons are meant to be interactive. A non-interactive button is confusing. Both for mouse users and non-mouse (e.g. keyboard) users, but especially for the latter.

I think using the emojis as a key would help. Will comment in the other thread.

Reply:
ok, i can replace the single letter with an emoji, but then I still think it runs into the same problem.
in other words, an emoji is just "normal text", so how do we indicate to the end-user that this particular emoji has some special mouseover text functionality?

Reply:
> I think you can add title to other elements too

Yes, as I said above, I can add `title` to e.g. `p` or `div` element, but then there is no visual indication that mouseover functionality exists, so we need a solution to that problem. (changing the cursor when mousing over doesn't solve the problem because you have to mouse over it first and you would have no reason to do that in the first place)

Reply:
> I think it would make sense to remove package name from the rule names.

I kind of like the full name for copy-pasting purposes from the docs into a e.g. eslint-disable comment, and then also for the inverse e.g. ctrl+f for a rule that was triggered in my codebase without manually having to mutate the text.

what does josh think about this?

Reply:
which Josh?

Reply:
Just kidding - I'm still ruminating. It's a surprisingly hard UX issue...!

Reply:
> Is it possible to hide the plugin name if screen is narrow?

i vote no to keep it simple

Reply:
the current version of the PR has emojis, but the outstanding issue is that the emojis need to have some signifier that they have mouseover functionality (or the table could be reworked in some other way).

Reply:
a problem for future us